_templates/chain/new/default-entry.ejs.t

```diff
@@ -0,0 +1,39 @@
+---
+# helper funcs https://www.hygen.io/docs/templates/#helpers-and-inflections
+to: blocks/<%= h.inflection.dasherize(block_name) %>-block/chains/<%= h.inflection.dasherize(block_name) %>/default.jsx
+---
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+import { Heading, HeadingSection } from '@wpmedia/shared-styles';
```

Comment:
The generator for chains is likely to be used for new enhanced styling work - I am a little nervous having this generator reference the shared styles block.
We should make a note to fix this in the near future or remove this from the current generator

Reply:
hm ok. which way would you lean "We should make a note to fix this in the near future" or "remove this from the current generator"? 

Since we don't have enhanced styling work live yet, I tend to lean toward doing this how we do shared-styles now. then, when we fix the architecture for the existing blocks, we fix it. in this way, the generator reflects the current architecture. 

If we go the opposite way, then the chain generator would be fairly bare. Just rendering children, right? I can certainly do that. but we'd still need to go back and fix this when enhanced styling comes around, I think

Reply:
if you're nervous about this implementation, it might be better to keep it minimal and go with "remove this from the current generator". I can move forward with that for now

Reply:
updated @matthewroach